sdk/spring/azure-spring-data-cosmos/src/test/java/com/azure/spring/data/cosmos/domain/BasicItem.java

```diff
@@ -0,0 +1,51 @@
+package com.azure.spring.data.cosmos.domain;
+
+import com.azure.spring.data.cosmos.core.mapping.Container;
+import com.azure.spring.data.cosmos.core.mapping.PartitionKey;
+import org.springframework.data.annotation.Id;
+
+import java.util.Objects;
+
+@Container()
+public class BasicItem {
+
+    @Id
+    @PartitionKey
+    private String id;
```

Comment:
Add another type for testing where the name of this field is something other than "id" to ensure everything is working properly.

Reply:
The @Id notation here is actually redundant/misleading, and I should have left it out. The “id” field will be defined as Id by default. It doesn’t matter what we define as id, only whether the id field is defined as @partitionKey